analysis_options.yaml

```diff
@@ -5,6 +5,9 @@ analyzer:
     strict-casts: true
     strict-inference: true
     strict-raw-types: true
+  errors:
+    # analyzer deprecations
```

Comment:
What version are they deprecated in? I don't see any warnings with the latest published, do I need a git override to repro?

Reply:
Nevermind, I'm just not getting diagnostics from the anlaysis server in this repo.

Reply:
there are very much deprecation warnings w/ the latest stable release!